### ISSUE TITLE
Adding new prop to disable toggle

### ISSUE THREAD

**Issue by johanjmoncada** [react-native-seoul/react-native-switch-toggle#7]
The original issue was that despite I don't change de value to switchOn the animation is performed at least one time, so that I've just add a custom prop to disable animation when we want to not change the toggle, I could push the code if the owner of this repo allow me.

**Comment by rollsroyc3** [comment]
Can you create a PR as I just ran into this issue also?

**Comment by dooboolab** [comment]
@johanjmoncada Yes. Please share your work with `PR`. I just can't handle everything since I am holding some other repos too.

**Comment by SergioGeeK7** [comment]
I was searching for the same thing, try wrapping the component like this when you want to set it as  disabled

<View pointerEvents="none">
  <SwitchToggle />
</View>

**Comment by rollsroyc3** [comment]
https://github.com/dooboolab/react-native-switch-toggle/pull/9

**Comment by rollsroyc3** [comment]
Theres a good chance this will screw up legacy users who expect it to change right away, and/or might not be passing a value prop but just listening to the onPress event.

**Comment by SergioGeeK7** [comment]
well, they should have locked package versions anyway, `yarn.lock` or similar
